WordPress/UITestsFoundation/Screens/ActivityLogScreen.swift

```diff
@@ -2,12 +2,36 @@ import ScreenObject
 import XCTest
 
 public class ActivityLogScreen: ScreenObject {
+    public let tabBar: TabNavComponent
+
+    let dateRangeButtonGetter: (XCUIApplication) -> XCUIElement = {
+        $0.buttons["Date Range"].firstMatch
+    }
+
+    let activityTypeButtonGetter: (XCUIApplication) -> XCUIElement = {
+        $0.buttons["Activity Type"].firstMatch
+    }
+
+    var dateRangeButton: XCUIElement { dateRangeButtonGetter(app) }
+    var activityTypeButton: XCUIElement { activityTypeButtonGetter(app) }
 
     public init(app: XCUIApplication = XCUIApplication()) throws {
+        tabBar = try TabNavComponent()
+
         try super.init(
-            expectedElementGetters: [ { $0.otherElements.firstMatch } ],
+            expectedElementGetters: [ dateRangeButtonGetter, activityTypeButtonGetter ],
             app: app,
             waitTimeout: 7
         )
     }
+
+    public static func isLoaded() -> Bool {
+        (try? ActivityLogScreen().isLoaded) ?? false
+    }
+
+    @discardableResult
+    public func verifyActivityLogScreenLoaded() -> Self {
+        XCTAssertTrue(ActivityLogScreen.isLoaded(), "\"Activity\" screen isn't loaded.")
```

Comment:
For the assert, the test is asserting the top bars (date range and activity buttons) and not the items displayed on the page (in this case, what's displayed on the card and on the screen itself are completely different - i guess this is because of mocks) is this expected?

Reply:
The reasoning here is the as same in [this comment](https://github.com/wordpress-mobile/WordPress-iOS/pull/20788#discussion_r1218073023). 

> what's displayed on the card and on the screen itself are completely different - i guess this is because of mocks

Correct, the mock file for Activity Log screen existed before, and I did not touch it. LMK if you think it makes sense to update it, for consistency.

Reply:
Well, the comment from above did not age well, too. I've updated the mock file for activity log screen, and also now checking that the same activities are present in the final screen. See f9746e63a71550d0621a57e198a8644f21e922f7.